### ISSUE TITLE
"indices.filter_cache" cluster statistic not present

### ISSUE THREAD

**Issue by multani** [elastic/elasticsearch#16226]
I'm using an external monitoring system which collects metrics from Elasticsearch and tries to collect the ``indices.filter_cache`` values as shown in https://www.elastic.co/guide/en/elasticsearch/reference/2.1/cluster-stats.html:
```
{
   "indices": {
...
      "filter_cache": {
         "memory_size": "0b",
         "memory_size_in_bytes": 0,
         "evictions": 0
      },
...
```
For some reason, I don't have these values when I try to query the different clusters I have (running Elasticsearch 2.1 with ``cloud-aws`` as the only configured plugin). Grepping the code doesn't shown any relevant information about this value, so I wonder where the problem is?
Is this a value which is returned by a plugin (which I don't have enabled)? Is this an obsolete value? Is this activated only under some specific circumstances?

**Comment by clintongormley** [comment]
The filter cache was renamed to the query cache in 2.0.0 - I've updated the docs to reflect that

**Comment by clintongormley** [comment]
Closed in 4e5316591a95b28528335e1905558bfa1c0c4c6a
